### PR TITLE
Simplify the consulting-software engineer post

### PR DESCRIPTION
**Pull request by ctro**
@rooneywp here's this PR again, w/o all the extra cruft.

### CODE REVIEW

File: pages/positions/consulting-software-engineer.md [hunk 1]
```diff
@@ -27,8 +27,14 @@ supervisory_status: 'No'
 ---
 
 {% if page.state == 'upcoming' %}
-{{ page.org }} will soon be accepting applications for a GS-{{ page.gs_level }} {{ page.title }}.
-{% if page.opens == 'tbd' %} The target date for when this position will be officially open to application has not yet been determined. If you'd like to be notified when this position is open, sign up to our [mailing list](https://public.govdelivery.com/accounts/USGSATTS/subscriber/new?topic_id=USGSATTS_6). {% else %} Applications will be open for submission on {{ page.opens | date: '%l:%M%P %Z on %A, %B %e, %Y' }}. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process. 
+  {{ page.org }} will soon be accepting applications for a GS-{{ page.gs_level }} {{ page.title }}.
+  {% if page.opens == 'tbd' %} The target date for when this position will be officially open to application has not yet been determined. If you'd like to be notified when this position is open, sign up to our [mailing list](https://goo.gl/forms/QMbyPse8f4rfnZ9z2).
```
Comment (ctro) [inline pages/positions/consulting-software-engineer.md]: Notice the whitespace in front of the 2nd if statement -- i think that is requried, and might be why you were having problems w/ nested if statements.

Comment (rooneywp) [reply pages/positions/consulting-software-engineer.md]: Ah, yeah that could have been it.

File: pages/positions/consulting-software-engineer.md [hunk 2]
```diff
@@ -27,8 +27,14 @@ supervisory_status: 'No'
...
+  {% else %} Applications will be open for submission on {{ page.opens | date: '%l:%M%P %Z on %A, %B %e, %Y' }}. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process.
+  {% endif %}
+{% endif %}
+
+{% if page.state != 'upcoming' %}
```
Comment (rooneywp) [inline pages/positions/consulting-software-engineer.md]: When I was reviewing your changes I also realized that these if statements can just be used to insert things. I think I was also stuck on the idea that it was more like branching logic so all paths had to be defined.

**Comment by rooneywp** [comment]
@ctro I pushed a commit to this branch. I moved the sections around so it lined up with the required format when set to `open'. The other changes where things that vsc was showing me as markdown issues, like trailing spaces.

**Comment by ctro** [comment]
@rooneywp this looks good to me!
Nice work.
Also thanks for leaving things better than you found them w/ the Markdown formatting!

The only potential 🤔 is that the "Basic Info" section is pretty far down the listing.  I feel like people might be looking for that info at the top?

File: pages/positions/consulting-software-engineer.md [hunk 3]
```diff
@@ -27,19 +27,23 @@ supervisory_status: 'No'
...
+  {% else %} Applications will be open for submission on {{ page.opens | date: '%l:%M%P %Z on %A, %B %e, %Y' }}. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process.
+  {% endif %}
 {% endif %}
 
-## Basic Information
+{% if page.state != 'upcoming' %}
+  {{ page.org }} is hiring for GS-{{ page.gs_level }} {{ page.title }}. This page contains information related to the role as well as a link to submit your application. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process.
+{% endif %}
 
 **Location:**
```
Comment (rooneywp) [inline pages/positions/consulting-software-engineer.md]: @ctro The issue is that we have to match the required job announcement format, which requires the `Basic Information` to be where it is. This isn't a problem when it's just upcoming, which is why we put that info at the top. But once it's `open` and is an official job announcement, we have to adhere to the format. One thing we could do is that I've done here. Duplicate the `Location`. `Salary Range` and `Who May Apply` sections and always keep them at the top.

File: pages/positions/consulting-software-engineer.md (hunk 2 above)
Comment (ctro) [reply pages/positions/consulting-software-engineer.md]: Right!  You don't have to follow every "if" with any other conditions or an "else".  You can just use a quick `if`/`endif` statement to, maybe, display a banner when the post if live.

File: pages/positions/consulting-software-engineer.md (hunk 3 above)
Comment (ctro) [reply pages/positions/consulting-software-engineer.md]: :+1: OK, gotcha.  In that case I think duplicating these pieces makes sense.

**Pull request merged**